### DISCUSSION TITLE
Monitor input method changes via dbus-monitor

### DISCUSSION THREAD

**Discussion by andrhua** [fcitx/fcitx5#829] [Q&A]
I'm trying to get [polybar-fcitx5](https://github.com/RRRRRm/polybar-fcitx5-script/blob/main/polybar-fcitx5-script.sh#L83) script working, everything seems to be operating fine except for the `dbus-monitor`, it shows no events when changing layouts (I have colemak, ru, and hangul configured). I tried to poke a little with no success. I thought notification tray addon may help there but it seems it subscribes to internal events, not dbus ones.

**Comment by Bowen951209** [comment]
@andrhua Have you solved it?
